ui/client/components/graph/node-modal/editors/expression/StringEditor.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import RawEditor from "./RawEditor"
+
+export default function StringEditor(props) {
+
+  const trim = (value) => value.substring(1, value.length - 1)
+
+  const format = (value) => quotationMark + value + quotationMark
+
+  return <RawEditor {...props} trimValue={trim} formatValue={format}/>
+}
+
+export const parseableString = (expressionObj) => {
+  const expression = expressionObj.expression
+  const language = expressionObj.language
+  return stringPattern.test(expression) && language === "spel"
+}
+
+const stringPattern = /^'.*'$/
```

Comment:
What about expression:   'first_part' + #input.myVar + 'second_part'? And what about "?

Reply:
I see now that is a bit more complicated than we thought. 
Two questions:
1. Any phrase which is not in '...' should start with # ? 
2. Are "..." valid replacement for '...'?

Reply:
"" is supported now